history.adoc

```diff
@@ -1,12 +1,13 @@
 :issues: https://github.com/cf-convention/cf-conventions/issues/
 :pull-requests: https://github.com/cf-convention/cf-conventions/pull/
-:tickets: http://cfconventions.org/Data/Trac-tickets/
+:tickets: https://cfconventions.org/Data/Trac-tickets/
```

Comment:
This link (also) appears to be broken, leading to a 'Not Found' page regardless of the protocol used (HTTP or HTTPS).

Reply:
I guess this is the right link https://cfconventions.org/Data/trac.html

Reply:
Indeed, that looks right. )The original referenced link was probably right once-upon-a-time but has since been updated due to some restructure.)

Reply:
Updated to the above in [6cb0d99](https://github.com/cf-convention/cf-conventions/pull/476/commits/6cb0d99cbb119ef31b5da779bab321f8f7f90a83), so marking this as resolved.